lib/builtins/deploy-delegates/cfn-deployer/helper.js

```diff
@@ -99,6 +100,11 @@ function deployStack(reporter, options, callback) {
             }
         ]
     };
+
+    // merge custom deployer parameters with standard parameters
+    // custom deployer parameters can overwrite standard parameters
+    stackOptions.stackParameters = R.unionWith(R.eqBy(R.prop('ParameterKey')), deployerParameters, stackOptions.stackParameters);
```

Comment:
if we dont want standard params to be overwritten, we should switch the order of union.